From 73bc139373d72bc9714510af3da833eeb7427d89 Mon Sep 17 00:00:00 2001
From: Aparna Mallavarapu <aparnam@codeaurora.org>
Date: Fri, 12 Oct 2012 18:44:41 +0530
Subject: [PATCH 07/75] input: Makefile: Add support for QRD SKUD

During the merge of msm-3.4, the TS driver got ommitted
from compilation. The same is added here for QRD SKUD.

Change-Id: I511c2856310595d9e1ad94c1247d168ec40312ca
Signed-off-by: Aparna Mallavarapu <aparnam@codeaurora.org>
---
 drivers/input/touchscreen/Makefile |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/input/touchscreen/Makefile b/drivers/input/touchscreen/Makefile
index 5a1eec7..7b28e9d 100644
--- a/drivers/input/touchscreen/Makefile
+++ b/drivers/input/touchscreen/Makefile
@@ -79,3 +79,4 @@ obj-$(CONFIG_TOUCHSCREEN_TPS6507X)	+= tps6507x-ts.o
 obj-$(CONFIG_TOUCHSCREEN_MSM_LEGACY)		+= msm_touch.o
 obj-$(CONFIG_TOUCHSCREEN_CY8C_TS)	+= cy8c_ts.o
 obj-$(CONFIG_TOUCHSCREEN_CYTTSP_I2C_QC)       += cyttsp-i2c-qc.o
+obj-$(CONFIG_TOUCHSCREEN_FT5X06)	+= ft5x06_ts.o
-- 
1.7.0.4

